frontend/src/assets/translations/en.json

```diff
@@ -0,0 +1,30 @@
+{
+  "about": "About",
+  "volunteers": "Volunteers",
+  "contactus": "Contact Us",
+  "signup": "Sign Up",
+  "login": "Log In",
+  "findyourvolunteer": "FIND YOUR VOLUNTEER",
+  "volunteerswhocanhelp": "VOLUNTEERS WHO CAN HELP",
+  "skill": "Skill",
+  "city": "City",
+  "language": "Language",
+  "whoarewe": "WHO ARE WE?",
+  "aboutbehelp1": "BeHelp is a service that connects refugees and asylum seekers to local volunteers who are willing to share their expertise and skills pro bono.",
+  "aboutbehelp2": "On BeHelp, you can find helpers from all around Belgium.",
+  "ourservices": "Our Services",
+  "ourservices2": "BeHelp is a group of volunteers who have the time, desire and opportunity to help newcomers to Belgium with everyday problems.",
+  "hyfstudents": "Project from the students of HYF",
+  "kidane": "BeHelp made me believe again. After fleeing from Eritrea, I had a very hard time and needed psychological help which I could hardly afford. On this platform, I could find a volunteer psychotherapist that renders aid to people like me. She even spoke Tigrinya! Our sessions allowed me to see new opportunities and live a more fulfilling life, even though it remains challenging to adapt to the new country.",
+  "levi": "The drama of refugees fleeing out their countries has always touched me, also because my father was a refugee once! Before, I was simply donating money to fundraising movements, and BeHelp gave me an excellent opportunity to contribute in a more personal way. Being a French teacher, I give language classes and serve as an interpreter from Arabic, the language that I master from my childhood.",
+  "abdel": "Having left my country behind, I faced extreme complexity of the Belgian legal system. All these papers and offices … hard to deal with if you don’t speak Dutch! So I registered on BeHelp and found competent volunteers that explained me how to smoothen the process of legalization. Now I am returning all support I have received here, by helping people who came to Belgium, like me, and need help of a professional psychologist.",
```

Comment:
"Nadat ik mijn land achter me had gelaten, werd ik geconfronteerd met de extreme complexiteit van het Belgische rechtssysteem. Al deze papieren en kantoren ... moeilijk om mee om te gaan als je geen Nederlands spreekt! Dus ik registreerde me op BeHelp en vond bekwame vrijwilligers die me uitlegden hoe het proces van legalisatie. In elkaar zit. Nu geef ik alle steun terug die ik hier heb ontvangen, door mensen te helpen die naar België kwamen, zoals ik, en hulp nodig hebben van een professionele psycholoog."